packages/rancher-sriov/generated-changes/exclude/crds/k8s.cni.cncf.io_networkattachmentdefinitions_crd.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 NVIDIA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: network-attachment-definitions.k8s.cni.cncf.io
```

Comment:
I see `network-attachment-definition` is listed in the clusterrole `sriov-network-operator`. 
It is ok to include it in the clusterrole even the CRD does not exist. I just need to confirm that we do want to exclude this CRD.

Reply:
SR-IOV requires Multus. Multus allows to set up multiple interfaces in a pod. Multus becomes the main CNI plugin and can propagate calls to different "real" CNI plugins that are registered. The way to register those CNI plugins is through the crd: network-attachment-definitions.k8s.cni.cncf.io. However, that CRD is part of Multus design and thus it should be installed by Multus, not by SR-IOV. In fact, we added a few lines to detect that and if the crd is not installed, then we ask the user to install Multus first: https://github.com/rancher/charts/pull/1211/files#diff-c0260e86c59f0bb92c1629b68aad1d2f97629b9d4d1978783d810ce3470acb02R1. If we install that CRD via SR-IOV, then SR-IOV will start but we will not be able to properly allow pods to use it because we will be missing Multus binary and config which is installed by Multus helm chart. BTW, this is documented